src/monitor-pipeline-group/azext_monitor_pipeline_group/aaz/latest/monitor/pipeline_group/_update.py

```diff
@@ -0,0 +1,1287 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "monitor pipeline-group update",
+)
+class Update(AAZCommand):
+    """Update a pipeline group instance.
+    """
```

Comment:
please add example for update and create cmd in codegen tool

Reply:
Added example.